BadHabbit/BadHabbit/UserStories/SingnInAppleScreen/SignInAppleViewController.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  SignInAppleViewController.swift
+//  BadHabbit
+//
+//  Created by Halina Kurylchykava on 14.07.23.
+//
+
+import UIKit
+import SnapKit
+import AuthenticationServices
+
+class SignInAppleViewController: UIViewController {
+    private var authorizationButton: ASAuthorizationAppleIDButton!
+    private var imageLogo: UIImageView!
+    private var imageMountains: UIImageView!
+    private var titleLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+}
+
+extension SignInAppleViewController {
+
+    private func setupUI() {
+        view.backgroundColor = UIColor(red: 0.81, green: 0.81, blue: 0.81, alpha: 1.00)
+        imageMountains = createImageView(name: "mountains")
+        imageLogo = createImageView(name: "logo")
+        createAuthorizationButton()
+        createTitleLabel()
+        setupConstraints()
+    }
+
+    private func createImageView(name: String) -> UIImageView {
+        let imageName = name
+        let image = UIImage(named: imageName)
+        let imageView = UIImageView(image: image)
+        view.addSubview(imageView)
+
+        return imageView
+    }
+
+    private func createTitleLabel() {
+        titleLabel = UILabel()
```

Comment:
Do you have errors in console logs due to conflicting constraints?

Reply:
there are no messages in console log

Reply:
By default `translatesAutoresizingMaskIntoConstraints`is true so AutoLayout added constraints automatically. This constraints always conflict with your own constraints

Reply:
snapkit documentation says:
Ensuring translatesAutoresizingMaskIntoConstraints = false is called on all appropriate views.
Or do we still need to add it?

Reply:
You're right